pkg/apis/k0s.k0sproject.io/v1beta1/kuberouter.go

```diff
@@ -24,19 +24,32 @@ type KubeRouter struct {
 	MTU int `json:"mtu"`
 	// Kube-router metrics server port. Set to 0 to disable metrics  (default: 8080)
 	MetricsPort int `json:"metricsPort"`
-	// Activate Hairpin Mode (allow a Pod behind a Service to communicate to its own ClusterIP:Port)
-	HairpinMode bool `json:"hairpinMode"`
+	// Admits three values: "enabled" enables it globaly, "allowed" allows but services must be annotated explictly and "disabled"
```

Comment:
Some small corrections.

```suggestion
	// Admits three values: "Enabled" enables it globally, "Allowed" allows but services must be annotated explicitly and "Disabled".
```